src/main/java/gift/dto/ProductRequest.java

```diff
@@ -0,0 +1,6 @@
+package gift.dto;
+
+public record ProductRequest(Long id,
+                             String name,
+                             int price,
+                             String imageUrl) { }
```

Comment:
현재는 DB 에서 채번한 값을 가지고 Product 의 PK 가 정해지고 있어서 id 필드는 제거해도 될 것 같습니다😀

만약 DB 를 사용하지 않는다면 클라이언트로부터 PK 를 생성하는 것보다는 서버쪽에서 AtonomicLong 등의 threadsafe 한 PK 생성기롤 만들어보는 것도 좋을 것 같아요!

fyi; [AtomicLong 관련 문서](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/package-summary.html)
```suggestion
public record ProductRequest(String name,
                             int price,
                             String imageUrl) { }
```